client/src/components/order/new-order.js

```diff
@@ -234,6 +251,16 @@ class NewOrder extends Component {
             />
           </Form.Group>
 
+          <Form.Group>
```

Comment:
Don't show it input, if there aren't dishes. Or make it interactive (when user click on this input add dish)
![image](https://user-images.githubusercontent.com/6152066/80940878-9da5b300-8de9-11ea-851e-a2488fbfcc4f.png)

Reply:
It isn't diffucult to hide this, but as we want to show user that he can add dishes to his order, this input should be visible. I think we will add similar functionality in our app.

Reply:
For now it looks like input, but it is disabled. If you want to show it all time, you should add some description (message, alert) to explain what it is. Or you can add redirect to dishes page from this form
But you shouldn't show blank, disabled and useless input on page